docs/acr42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,68 @@
+This document follows the [arc42 template](https://arc42.org/).
+# Introduction and Goals
+
+This document describes LAPIS, which is a platform to give easy access to genomic sequence data alongside metadata of
+the sequenced probes. It is used to filter sequence data and return the result to the user through web access, so the
+user can develop his/her own evaluation of the data. The current development is based on two existing
+systems (LAPIS-api and SILO), which are developed for SarsCov2 viruses, and shall be extended to also include other
+virus species.
+
+The following goals have been established for this system:
+
+| Priority |                                                            |
+|----------|------------------------------------------------------------|
+| 1        | Include current functionality of LAPIS in the new approach |
```

Comment:
"current functionality of LAPIS" = very efficient querying of large data sets of genomic sequences. So I would rather write that down, since in my understanding, that is the main goal, and it's independent of the existing system. Maybe the exact formulation has to be sharpened a bit though.

I would maybe also reformulate the second prio item to something like "Provide the infrastructure such that it is easy for other researchers to setup an instance for data of other virus species".

@chaoran-chen Is "virus species" actually the correct term here? That will probably appear throughout the whole document, so we should align on a term here.